plugins/de.cau.cs.kieler.klighd.ui.view/src/de/cau/cs/kieler/klighd/ui/view/controller/ViewUpdateControllerFactory.java

```diff
@@ -320,7 +320,7 @@ public static AbstractViewUpdateController getNewInstance(final String controlle
                     mucm.idControllerMapping.get(controllerID);
             if (clazz != null) {
                 try {
-                    AbstractViewUpdateController newInstance = clazz.newInstance();
+                    AbstractViewUpdateController newInstance = clazz.getConstructor().newInstance();
```

Comment:
Same for all reflective new instances: the deprecation warning suggests to use getDeclaredConstructor()